src/peergos/gwt/emu/java/nio/file/StandardOpenOption.java

```diff
@@ -0,0 +1,7 @@
+package java.nio.file;
+
+public enum StandardOpenOption implements OpenOption {
```

Comment:
is there a way we can avoid all these superfluous classes? Like have an interface, with a java impl in server, and a native JS one?

Reply:
Used by shared (FileTreeNode) rather then server. So in short no.

Reply:
But they're new, so I think we can avoid it by moving the video thumbnail java stuff to a tpye in server.